platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/ReconnectProtocolResponder.java

```diff
@@ -60,11 +63,13 @@ public ReconnectProtocolResponder(
             final LatestSignedStateProvider latestSignedStateProvider,
             final ReconnectSettings settings,
             final ReconnectThrottle reconnectThrottle,
+            final FallenBehindManager fallenBehindManager,
```

Comment:
null annotations and checks are missing

Reply:
Annotated with `@NonNull` + a check is added.